SourceCode/Freight/ftt_fr_main.py

```diff
@@ -389,6 +389,10 @@ def solve(data, time_lag, iter_lag, titles, histend, year, domain):
                             data_dt['ZCET'][:, tech, c6ti['1 Price of vehicles (USD/vehicle)']] \
                             * (1.0 + data["ZCET"][:, tech, c6ti['15 Learning exponent']]
                             * dw[tech]/data['ZEWW'][0, tech, 0])
+                            
+            battery_cap_transport = data["BTTC"][:, :, c6ti['21 Battery capacity (kWh)']]
+            yearly_additions_freight = (battery_cap_transport * data["ZEWI"][:, :, None]) / 1e3  # Convert kWh to GWh
+            cumulative_capacity_freight = np.sum(yearly_additions_freight, axis=0)  # Summing across regions
```

Comment:
BTTC needs to be ZCET. BTTC is the cost matrix for the transport sector, rather than the freight sector.

Reply:
Oh right!

Reply:
Minor comment: you may want to improve the comment on 394: there are two conversions here, from kWh to GWh and from 1000s to 1s.

Reply:
That seems correct. The variable names still need updating however (as we're no longer computing yearly additions).

Reply:
Makes sense...